File: src/targets/r/httr/fixtures/cookies.r
library(httr)

<<<<<<< HEAD
url <- "https://httpbin.org/cookies"

response <- VERB("GET", url, content_type("application/octet-stream"), set_cookies(`foo` = "bar", `bar` = "baz"))
=======
url <- "http://mockbin.com/har"

response <- VERB("POST", url, content_type("application/octet-stream"), set_cookies(`foo` = "bar", `bar` = "baz"))
>>>>>>> upstream/master

content(response, "text")

File: src/targets/javascript/jquery/fixtures/cookies.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/cookies',
  method: 'GET',
=======
  url: 'http://mockbin.com/har',
  method: 'POST',
>>>>>>> upstream/master
  headers: {
    cookie: 'foo=bar; bar=baz'
  }
};

$.ajax(settings).done(function (response) {
  console.log(response);
});